libraries/Microsoft.Bot.Schema/Teams/OnBehalfOf.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Bot.Schema.Teams
+{
+    using Newtonsoft.Json;
+
+    /// <summary>
+    /// Specifies attribution for notifications.
+    /// </summary>
+    public partial class OnBehalfOf
```

Comment:
I think it would be a better idea to name it `TeamsMeetingNotificationOnBehalfOf` instead. Here are two reasons why:
1. It follows the convention of the other schema classes you have created for this feature.
2. `OnBehalfOf` object is not seen in any other Teams related schemas so having a generic name is not particularly useful.